client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -1,9 +1,10 @@
 import { StepContainer } from '@automattic/onboarding';
 import { useTranslate } from 'i18n-calypso';
 import { useState } from 'react';
+import { Onboard } from 'calypso/../packages/data-stores/src';
```

Comment:
The import reference should be via a package, i.e. '@automattic/data-store'.

Reply:
Both fixed!